tests/acceptance/stepDefinitions/webdavContext.js

```diff
@@ -18,33 +18,44 @@ const path = require('../helpers/path')
  */
 function fileExists (userId, element) {
   const headers = httpHelper.createAuthHeader(userId)
+  const davPath = webdavHelper.createDavPath(userId, element)
   return fetch(
-    webdavHelper.createDavPath(userId, element),
+    davPath,
     { method: 'GET', headers: headers }
   )
-    .then(function (res) {
-      if (res.status === 200) {
-        return res
-      } else {
-        throw Error('file/folder should exist, but does not')
-      }
-    })
 }
 
+const fileShouldExist = function (userId, element) {
+  return fileExists(userId, element).then(function (res) {
+    assert.strictEqual(res.status, 200, 'File should exist, but does not')
+  })
+}
+
+const fileShouldNotExist = function (userId, element) {
+  return fileExists(userId, element).then(function (res) {
+    assert.ok([404, 401].includes(res.status), 'file/folder should not exist, but does')
+  })
+}
+
+Then('as {string} file/folder {string} should not exist{opt_on_remote}', function (userId, element, remote) {
```

Comment:
Just tried this. If we do not like this custom parameter type, i could just drop it. Otherwise, i will refactor other steps in  a separate PR.

Reply:
cc'ing @individual-it @HariBhandari07 @kiranparajuli589 @jasson99 for opinions.

Reply:
I agree with @dpakach on this one.

Reply:
agree with @dpakach

Reply:
Okay, doing it on separate steps.